ocs_ci/ocs/resources/storageconsumer.py

```diff
@@ -705,16 +705,150 @@ def verify_storage_consumer_resources(
+        )
+        ceph_data_on_consumer_match_arg["csiop-cephfs-client-profile"] = (
+            ceph_data_arg.get("csiop-cephfs-client-profile", "")
+            == storage_consumer_uid_arg
+        )
+        ceph_data_on_consumer_match_arg["csiop-rbd-client-profile"] = (
+            ceph_data_arg.get("csiop-rbd-client-profile", "")
+            == storage_consumer_uid_arg
+        )
+
+    def _handle_storageconsumer_ceph_client_names_upgraded_converged(
+        ceph_data_arg, ceph_data_on_consumer_match_arg, storage_consumer_uid_arg
+    ):
+        """
+        Handle post-converged client names for ceph data on consumer match.
+
+        Args:
+            ceph_data_arg (dict): The ceph data dictionary from the config map.
+            ceph_data_on_consumer_match_arg (dict): The dictionary to store the match results.
+            storage_consumer_uid_arg (str): The UID of the storage consumer.
+        """
+
+        ceph_data_on_consumer_match_arg["csi-cephfs-node-ceph-user"] = (
+            ceph_data_arg.get("csi-cephfs-node-ceph-user", "")
+            == f"cephfs-node-{storage_consumer_uid_arg}"
+        )
+        ceph_data_on_consumer_match_arg["csi-cephfs-provisioner-ceph-user"] = (
+            ceph_data_arg.get("csi-cephfs-provisioner-ceph-user", "")
+            == f"cephfs-provisioner-{storage_consumer_uid_arg}"
+        )
+        ceph_data_on_consumer_match_arg["csi-rbd-node-ceph-user"] = (
+            ceph_data_arg.get("csi-rbd-node-ceph-user", "")
+            == f"rbd-node-{storage_consumer_uid_arg}"
+        )
+        ceph_data_on_consumer_match_arg["csi-rbd-provisioner-ceph-user"] = (
+            ceph_data_arg.get("csi-rbd-provisioner-ceph-user", "")
+            == f"rbd-provisioner-{storage_consumer_uid_arg}"
+        )
+        ceph_data_on_consumer_match_arg["csiop-cephfs-client-profile"] = (
+            ceph_data_arg.get("csiop-cephfs-client-profile", "")
+            == storage_consumer_uid_arg
+        )
+        ceph_data_on_consumer_match_arg["csiop-rbd-client-profile"] = (
+            ceph_data_arg.get("csiop-rbd-client-profile", "")
+            == storage_consumer_uid_arg
+        )
+
+    def _handle_converged_client_ceph_names(
+        ceph_data_arg, ceph_data_on_consumer_match_arg
+    ):
+        """
+        Handle post-converged client names for ceph data on consumer match.
+
+        Args:
+            ceph_data_arg (dict): The ceph data dictionary from the config map.
+            ceph_data_on_consumer_match_arg (dict): The dictionary to store the match results.
+
+        """
+        ceph_data_on_consumer_match_arg["csi-cephfs-node-ceph-user"] = (
+            ceph_data_arg.get("csi-cephfs-node-ceph-user", "") == "rook-csi-cephfs-node"
+        )
+        ceph_data_on_consumer_match_arg["csi-cephfs-provisioner-ceph-user"] = (
+            ceph_data_arg.get("csi-cephfs-provisioner-ceph-user", "")
+            == "rook-csi-cephfs-provisioner"
```

Comment:
Do you think it would make sense to re-use constants from https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L1971 ?